src/app/webserver/app-pools/app-pools.service.ts

```diff
@@ -140,7 +133,7 @@ export class AppPoolsService {
         this.stop(pool);
 
         if (pool.status == Status.Stopping) {
-            let ob = IntervalObservable.create(200).subscribe(i => {
+            let ob = interval(200).subscribe(i => {
```

Comment:
This looks like a legacy code. For safety, please check if we need to call unsubscribe or not.